res/lang/missions/missions_en.txt

```diff
@@ -28,10 +28,10 @@ MISSION_TITLE_SAMMY = Sammy
 
 MISSION_GOAL_MAXIMIZE = %GOAL% as high as possible
 MISSION_GOAL_LESSTHAN = always less than %GOAL%
-MISSION_GOAL_IMAGE    = %VALUE%% Image
+MISSION_GOAL_IMAGE    = %VALUE%% image
```

Comment:
I assume the "Upper Case Text"-style was used here because of being used like in a "heading/caption" or as "title of something". (in the changes below you also kept that style for "Room Names")

In my opinion both variants are possible.

Reply:
Actually looking at how it's used in practice in the game, I agree with the original capitalization.  I'll change it back.